participant/package.json

```diff
@@ -17,6 +17,7 @@
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "e2e": "npx playwright test --retries=3",
+    "e2e:update": "npx playwright test --update-snapshots",
```

Comment:
@microwavenby I added the command `npm run e2e:update` for us to be able to update the snapshots.